snuba/datasets/cdc/message_filters.py

```diff
@@ -31,9 +31,7 @@ def should_drop(self, message: Message[KafkaPayload]) -> bool:
         )
 
         if table_name:
```

Comment:
If there is no table name, this method falls through to `False`, as in "this message should not be dropped."

This means that CDC events that are not associate with a table (i.e. begin, commit) are sent to the worker pool and require serialization and deserialization on both ends of that parallel transformation.

I will filter these out in a separate change, and I would expect this filter the stream by up to 2/3rds given our substantial usage of autocommit.